templates/taxi/index.html

```diff
@@ -10,4 +10,8 @@ <h2>Dynamic content</h2>
     <li><strong>Drivers:</strong> {{ num_drivers }}</li>
     <li><strong>Manufacturers:</strong> {{ num_manufacturers }}</li>
   </ul>
+    <br>
+    {% if user.is_authenticated %}
+        <p>{{ user.first_name }} {{ user.last_name }} visit this site: {{ num_visit }} time{{ num_visit|pluralize }}</p>
+    {% endif %}
```

Comment:
if a user is not authenticated they will not see this page at all so this check is redundant

Reply:
remove this check